src/Development/IDE/Core/Compile.hs

```diff
@@ -451,6 +453,10 @@ getModSummaryFromImports
   -> Maybe SB.StringBuffer
   -> ExceptT [FileDiagnostic] m ModSummary
 getModSummaryFromImports fp contents = do
+    fileModTime <- liftIO $ getModificationTime fp `catch` \e ->
```

Comment:
Is there a reason why this is not using the `GetModificationTime` rule? I’m somewhat hesitant to introduce a `fileModTime` that will change even if the document version does not change for a virtual file.

Reply:
That's a good point, this should be using the rule. Will fix

Reply:
Sorted